src/azure-cli-testsdk/azure/cli/testsdk/base.py

```diff
@@ -214,6 +214,8 @@ def __init__(self, method_name):
         self.kwargs = {}
         self.test_resources_count = 0
 
+        patch_main_exception_handler(self)
```

Comment:
`ScenarioTest` (inherited from `ReplayableTest`) patches `azure.cli.core.util.handle_exception` so that the raw exception is thrown and can be captured by `assertRaises`. (`handle_exception` silences any exceptions.)

Do the same for `LiveScenarioTest`.